.pipelines/templates/arc/extension-create.yaml

```diff
@@ -45,6 +45,23 @@ steps:
         exit 1
       fi
 
+      # validate if monitoring pod is Running
+      echo "validating monitoring pod..."
+      for i in $(seq 1 10); do
+        monitoringPodStatus=$(kubectl get pods -n kube-system -l app=arc-monitoring -o jsonpath='{.items[0].status.phase}')
```

Comment:
can we use `kubectl wait` with conditions instead here to check the deployment is ready?